### ISSUE TITLE
BUG: Keyboard input for Flow hotkey gets blocked in Fullscreen

### ISSUE THREAD

**Issue by MakiRollDev** [Flow-Launcher/Flow.Launcher#2464]
### Checks

- [X] I have checked that this issue has not already been reported.

- [X] I am using the latest version of Flow Launcher.

### Problem Description

I have my keybindings on CTRL + Shift + A

When I press these keys while in a fullscreen application/ game, the keys all ge blocked (as if the inputs are never made).

### To Reproduce

1. Enable Ignore hotkeys in fullscreen mode
2. When pressing the hotkey combination, it gets completely blocked.

### Flow Launcher Version

1.16.2

### Windows Build Number

22621.2861

**Comment by deefrawley** [comment]
Hi. If you enable Ignore hotkeys in fullscreen mode then Flow is behaving exactly as it should. if this is toggled OFF and this is still happening then there may be an issue

**Comment by MakiRollDev** [comment]
Okay, interesting, that seems kind of counterproductive. I would have assumed that flow would just ignore the inputs and let the other applications on the PC handle them.

**Comment by deefrawley** [comment]
Sorry? I am confused what the issue is. Flow shortcuts will get blocked in fullscreen if you have the Ignore shortucts in fullscreen setting enabled.

What is the issue?

**Comment by MakiRollDev** [comment]
Okay, sorry if I was unclear (English is not my first language)

When I press my hotkeys in a full screen application I would expect said application to receive those inputs as if flow wasn't running (in the game where I noticed: Ctrl(Crouch) + Shift(Sprint) +A(Walk left)) but instead the last key pressed doesn't go through.

So my character doesn't walk left.

It seems that flow still eats up the input even tho it doesn't react to it.

**Comment by deefrawley** [comment]
Ah I see now. And if you pause the game and exit out of Flow the keys start activating in the game?
Do you have any other keyboard listening apps running such as Powertoys, Autohotkey, Espanso etc.?

**Comment by MakiRollDev** [comment]
Yes, exiting flow fixes the issue. Powertoys was running, but i tried it without and it did not change anything.

**Comment by zeek-zag** [comment]
Same issue happens with Flow's game mode enabled. It's not freeing the shortcut and alt+space just stops working in other applications (Krita/SAI2 in my case).
Exiting the launcher frees the shortcut but it doesn't really fix the real issue :)

**Comment by VictoriousRaptor** [comment]
https://github.com/thomaslevesque/NHotkey/issues/36

Looks like not possible with our hotkey lib.

**Comment by NoPlagiarism** [comment]
Should be `keep-fresh`, since this one depends on another issue in another repo

**Comment by danteasy** [comment]
Any updates?

**Comment by jjw24** [comment]
I am implementing a new hotkey library #3157, will resolve this as part of that work.

**Comment by UltraJohn** [comment]
I'm still experiencing this issue. Quite annoying have to close Flow Launcher to use my set hotkey for other fullscreen applications.

**Comment by Ubawesome** [comment]
This should be higher priority tbh, completely eating inputs is a pretty massive bug.
